File: src/iniciante/ConversaoDeTempo.java
package iniciante;

import java.util.Scanner;

public class ConversaoDeTempo {
    public static void main(String[] args) {
        
        Scanner input = new Scanner(System.in);

        int N = (int) input.nextDouble();
    
        int min = N / 60;
        int horas = min / 60;
        min %= 60;
        int segundos = N % 60;
        System.out.printf("%d:%d:%d\n", horas, min, segundos);
         
        input.close();

    }
}

File: src/iniciante/ConversaoDeTempo.kt
package iniciante

/*Leia um valor inteiro, que é o tempo de duração em segundos de um determinado
evento em uma fábrica, e informe-o expresso no formato horas:minutos:segundos.
Entrada
O arquivo de entrada contém um valor inteiro N.
Saída
Imprima o tempo lido no arquivo de entrada (segundos), convertido
para horas:minutos:segundos, conforme exemplo fornecido.*/

import java.util.*

fun main(args: Array<String>) {

    val entrada = Scanner(System.`in`)
    val N: Int = entrada.nextInt()

    var minutos = N / 60

    val horas = minutos / 60
    minutos %= 60

    val segundos = N % 60

    println("$horas:$minutos:$segundos")
}